playbooks/roles/shadowsocks/templates/instructions.md.j2

```diff
@@ -106,7 +106,8 @@ This should return a 301 Found response **not** a connection refused error.
 
 <a name="ios"></a>
 ### iOS ###
-1. Download [Shadowrocket](https://itunes.apple.com/us/app/shadowrocket/id932747118?mt=8) and launch it.
+1. Download [Shadowrocket](https://itunes.apple.com/us/app/shadowrocket/id932747118?mt=8) and launch it.    
+> **Note:** Shadowrocket is not available in the China App Store. Apple was forced to remove it due to Chinese Government regulations. If you are using the China App Store and cannot find Shadowrocket, try switching to another country's App Store first.
```

Comment:
Hmm... I think it's important to let people know that there *is* a workaround by just switching your App store to another country.

If you guys prefer the shorter than that's cool, can you go ahead and edit my PR?